docs/cheaha/open_ondemand/ood_jupyter.md

```diff
@@ -40,9 +40,17 @@ For more information on GPU efficiency please see [Making the Most of GPUs](../s
     April 21, 2025: Currently, GPU-core affinity is not considered for GPU jobs on interactive apps. This may mean selecting multiple GPUs results in some GPUs not being used.
 <!-- markdownlint-enable MD046 -->
 
-## Extra Jupyter Arguments
+For more information on GPU efficiency please see [Making the Most of GPUs](../slurm/gpu.md#making-the-most-of-gpus).
+
+<!-- markdownlint-disable MD046 -->
+!!! important
+
+    April 21, 2025: Currently, GPU-core affinity is not considered for GPU jobs on interactive apps. This may mean selecting multiple GPUs results in some GPUs not being used.
```

Comment:
Good start. Improve it by offering a remedy. Suggest they should only request 1 GPU, or they can use HPC Desktop. I assume we have a way for them to start Jupyter in HPC Desktop, somewhere? If not, then don't worry about adding that to this PR, you can add an issue and someone can address that PR later.

This all applies only to amperenodes, so please clarify that.